changelog/T/TaxTaxation.md

```diff
@@ -0,0 +1,14 @@
+# What Has Changed In the “TaxTaxation” Article
+
+## Major changes
+Some words or sentences were changed because they seemed too complicated to the general understanding of the target audience.
+
+The word 'booty' was replaced by the word 'gain':
+ 
+ In this more stable period, income came from tribute but not from booty. --> In this more stable period, income came from tribute but not from gain.
```

Comment:
"gain" doesn't give the idea of stealing from someone. I'd suggest a word like "looting" -> ..."came from tribute but not from looting'